### PR TITLE
Created employee list

### PR DESCRIPTION
**Pull request by SindhuGouru**
Created employee list

### CODE REVIEW

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 1]
```diff
@@ -19,4 +22,14 @@
       return new ResponseEntity<>("Employee created successfully",HttpStatus.CREATED);
     }
 
+    @PostMapping("/createEmployeeList")
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: I would call this as createEmployees. If you change it please change the name of method too

File: employeeapplication/src/main/java/com/artimes/employee/model/Employee.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
 package com.artimes.employee.model;
 
 public class Employee {
-    public String empId;
-    public String empName;
-    public int empSalary;
+    private String empId;
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/model/Employee.java]: Goodjob!

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 3]
```diff
@@ -19,4 +22,14 @@
...
+    public ResponseEntity<String> createEmployeeList(@RequestBody EmployeeList employeeList){
+       for(int i=0;i<employeeList.getEmployeeList().size();i++) {
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: For each loop is much clearer to understand. Please refactor.

File: employeeapplication/src/main/java/com/artimes/employee/model/EmployeeList.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+    public List<Employee> getEmployeeList() {
+        return employeeList;
+    }
+
+    public void setEmployeeList(List<Employee> employeeList) {
+        this.employeeList = employeeList;
+    }
+
+    public List<Employee> employeeList;
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/model/EmployeeList.java]: Pojo variables should be private!

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java [hunk 5]
```diff
@@ -19,4 +22,14 @@
...
+           System.out.println(employeeList.getEmployeeList().get(i).getEmpId());
```
Comment (gourusumanth) [inline employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: If you wanna print all the fields I would rather use employee.toString() !!

File: employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java (hunk 1 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java]: Changed as requested

File: employeeapplication/src/main/java/com/artimes/employee/model/EmployeeList.java (hunk 4 above)
Comment (SindhuGouru) [reply employeeapplication/src/main/java/com/artimes/employee/model/EmployeeList.java]: modified

**Review by gourusumanth — APPROVED** [review]

**Pull request merged**